Library/Homebrew/completions/fish.erb

```diff
@@ -111,15 +111,21 @@ function __fish_brew_suggest_formulae_all -d 'Lists all available formulae with
     if test -f "$__brew_cache_path/descriptions.json"
         __fish_brew_ruby_parse_json "$__brew_cache_path/descriptions.json" \
             '.each{ |k, v| puts([k, v].reject(&:nil?).join("\t")) }'
+    else if test -n "$HOMEBREW_NO_INSTALL_FROM_API"
+        command ls -1 (brew --prefix)/Library/Taps/homebrew/homebrew-core
+    else if test -f "$__brew_cache_path/api/formula_names.txt"
+        command cat $__brew_cache_path/api/formula_names.txt
     else
+        # fallback on brew formulae if homebrew-core is not tapped
```

Comment:
Will this ignore non-core formulae after this change?

Reply:
Good question.